dev/doc/xml-protocol.md

```diff
@@ -753,3 +754,41 @@ Ex: `status = "Idle"` or `status = "proof: myLemmaName"` or `status = "Dead"`
 </feedback>
 ```
 
+## <a name="highlighting">Highlighting</a>
+
+Proof diffs, which are enabled by the `Diffs` option, highlight differences
+between the current and previous proof states in the displayed output.
+These are represented in output fields in the XML protocol with embedded
+XML-like tags.
+
+There are 4 tags that indicate how the enclosed text should be highlighted:
+- diff.added - added text
+- diff.removed - removed text
+- diff.added.bg - unchanged text in a line that has additions ("bg" for "background")
+- diff.removed.bg - unchanged text in a line that has removals
+
+CoqIDE, Proof General and coqtop currently use 2 shades of green and 2 shades of red
+as the background color for highlights.  Coqtop also applies underlining and strikeout
+highlighting for the sake of the color blind.
```

Comment:
CoqIDE does that too.
And is it not considered better to say "color blind people" rather than "the color blind" BTW?

Reply:
Grammatically it is fine.  Referring to "the disabled", "the homeless" etc is common.  But I can only think of examples referring to people; you would use "the burned ones" to refer to burned cookies, but not "the burned".

Or do you think it's insensitive?

Reply:
> Or do you think it's insensitive?

Right. Personally, I am not sure whether it is insensitive or not, but this was my worry.